std_msgs/Header header

float32 pos_x
float32 pos_y
float32 pos_z

uint16 range_value
int16 vrel_rad
int16 az_ang0
int16 az_ang1
int16 az_ang

int16 el_ang
int16 rcs0
int16 rcs1
int16 rcs

uint8 prob0
uint8 prob1
uint16 range_var
uint16 vrel_rad_var
uint16 az_ang_var0
uint16 az_ang_var1
uint16 az_ang_var

uint16 el_ang_var
uint8 pdh0
uint8 snr